src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,31 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { FC } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: FC<Props> = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article
+      className={cn(
+        'TodoInfo',
+        {
+          'TodoInfo--completed': completed,
+        },
+      )}
+    >
```

Comment:
Okay, it's interesting solution.
But to use a whole external library in order to insert 1 style ? :)
```suggestion
    <article className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`} >
```